src/utils/getFullYear.ts

```diff
@@ -0,0 +1,27 @@
+import { WEEK_LENGTH } from '../constants';
+
+const date = new Date();
+
+export default function getFullYear(monthOffset = 0, weekOffset = 0) {
+  if (weekOffset && monthOffset) {
+    return new Date(
+      date.getFullYear(),
+      date.getMonth() + monthOffset,
+      date.getDate() + weekOffset * WEEK_LENGTH
+    ).getFullYear();
+  }
+  if (monthOffset !== 0) {
+    return new Date(
+      date.getFullYear(),
+      date.getMonth() + monthOffset
+    ).getFullYear();
+  }
+  if (weekOffset !== 0) {
+    return new Date(
+      date.getFullYear(),
+      date.getMonth(),
+      date.getDate() + weekOffset * WEEK_LENGTH
+    ).getFullYear();
+  }
```

Comment:
I'm thinking that maybe these two if statements (line 13 and line 19) can be combined to one? Because if monthOffset/weekOffset equals to 0, then for example `date.getMonth()` and `date.getMonth + monthOffset` would return same result